dev/MRTCore/mrt/Core/unittests/UnitTest.vcxproj

```diff
@@ -135,6 +135,12 @@
       <SubSystem>Windows</SubSystem>
       <AdditionalLibraryDirectories>$(VCInstallDir)UnitTest\lib;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
     </Link>
+    <PostBuildEvent>
+      <Command>
+        copy $(TargetDir)\..\MRM\mrm.dll $(TargetDir)
```

Comment:
$(TargetDir)..\mrm\mrm.dll?

Reply:
I prefer this one as the other looks weird:)

Reply:
luckily \\ is tolerated in VS.

Reply:
Yep, we actually have that in a bunch of places. If you follow the entire compile spew you see a number of //s

---
In reply to: [489590532](https://github.com/microsoft/ProjectReunion/pull/189#discussion_r489590532) [](ancestors = 489590532)

Reply:
unfortunately project reunion's directory.build.props file sets a bad example.

Reply:
On second thought I am going to fix it. Let's not propagate incorrect things.